core/src/main/java/org/apache/iceberg/jdbc/JdbcUtil.java

```diff
@@ -53,21 +58,19 @@ final class JdbcUtil {
           + " WHERE "
           + CATALOG_NAME
           + " = ? AND "
-          + TABLE_NAMESPACE
-          + " = ? AND "
-          + TABLE_NAME
-          + " = ? AND "
+          + " %s = ? AND " // table or view namespace
```

Comment:
nit: we can remove the extra space before %s

Reply:
Indeed, I will also clean the existing SQL statements (which also have extra spaces). Thanks.